src/utils/errorArt.go

```diff
@@ -24,7 +24,7 @@ import (
 
 func ErrorMsg() {
 	rand.Seed(time.Now().UnixNano())
-	num := rand.Intn(5)
+	num := rand.Intn(6)
```

Comment:
You will probably have to change this to 7 instead of 6 for 6 different function calls, since `rand.Intn(n)` gives a pseudorandom number in the range [0, n) (n not included).

Reply:
Fixed in newest commit.